### PR TITLE
USAGOV-1042-published-pages-export: Created report, hook to modify level output to be readable

### PR DESCRIPTION
**Pull request by jacobaaronyeager**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1042

## Checklist for the Developer

- [ ] A link to the JIRA ticket has been added above.
- [ ] The JIRA ticket identifies the desired result of this change.
- [ ] The JIRA ticket contains clear acceptance criteria.
- [ ] The JIRA ticket contains clear testing steps.
- [ ] The JIRA ticket contains a description of "Done".
- [ ] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

**Comment by jacobaaronyeager** [comment]
there are some updates to the cms dockerfile folded into this due to my local failing entirely without them. This was left in because there are changes to the composer files that are for that as well as for this ticket, and sorting them was more of a pain than I wanted to get into.  I've tested it and it should be fine upstream, just fyi.

**Review by akf — CHANGES_REQUESTED** [review]
I ran into a problem with this one. When I click the little CSV button, I get a 500 error. This trace appears in my log: 
```
"PHP message: ArgumentCountError: SplFileObject::fputcsv() expects at most 4 arguments, 5 given in /var/www/vendor/league/csv/src/Writer.php on line 116 #0 /var/www/vendor/league/csv/src/Writer.php(116): SplFileObject->fputcsv(Array, ',', '"', '\\', '\n')
cms       | #1 /var/www/vendor/league/csv/src/Writer.php(122): League\Csv\Writer->League\Csv\{closure}(Array)
cms       | #2 /var/www/web/modules/contrib/csv_serialization/src/Encoder/CsvEncoder.php(162): League\Csv\Writer->insertOne(Array)
cms       | #3 /var/www/vendor/symfony/serializer/Encoder/ChainEncoder.php(40): Drupal\csv_serialization\Encoder\CsvEncoder->encode(Array, 'csv', Array)
cms       | #4 /var/www/vendor/symfony/serializer/Serializer.php(306): Symfony\Component\Serializer\Encoder\ChainEncoder->encode(Array, 'csv', Array)
cms       | #5 /var/www/vendor/symfony/serializer/Serializer.php(129): Symfony\Component\Serializer\Serializer->encode(Array, 'csv', Array)
cms       | #6 /var/www/web/modules/contrib/views_data_export/src/Plugin/views/style/DataExport.php(263): Symfony\Component\Serializer\Serializer" while reading response header
```

I saw some new error messages when I ran bin/build to bring in the new drupal modules, and again after I blew away my existing images and ran bin/init, so if this error doesn't occur for you we could look into that.

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
         "drupal/content_moderation_notifications": "^3.4",
         "drupal/core-composer-scaffold": "9.5.9",
         "drupal/core-project-message": "9.5.9",
-        "drupal/core-recommended": "9.5.9",
+        "drupal/core-recommended": "9.5.10",
```
Comment (akf) [inline composer.json]: As long as we're upgrading this to 9.5.10, I think it makes sense to upgrade the rest of drupal to 9.5.10. According to https://www.drupal.org/project/drupal/releases/9.5.10 it's a bug-fix release. I ran `bin/composer require drupal/core-recommended:9.5.10 drupal/core-composer-scaffold:9.5.10 drupal/core-project-message:9.5.10 --update-with-all-dependencies` (as recommended there, but with "bin/" and that seems like the way to go.

File: .circleci/config.yml [hunk 2]
```diff
@@ -540,6 +540,7 @@ workflows:
                 - dev
                 - stage
                 - prod
+                - USAGOV-1042-published-pages-export
```
Comment (akf) [inline .circleci/config.yml]: We'll want to remove these lines before merging.

**Review by akf — CHANGES_REQUESTED** [review]
All that remains to be done here is to remove the feature branch from the CircleCI config.

**Review by akf — APPROVED** [review]
Approving but not merging because I see requests for changes on the Jira ticket. We might decide to merge this anyway, though.

**Comment by akf** [comment]
Closing this PR because of #1303.

**Pull request closed without merging**